maplab-launch/launch/lidarstick/lidarstick-maplab-node-w-okvis.launch

```diff
@@ -0,0 +1,48 @@
+<?xml version="1.0"?>
+
+<launch>
+	<node pkg="rosbag" type="play" name="rosbag" args="/home/mfehr/experimental_ws/maplab/lidarstick_gonzen/gonzen_run_9_2019-02-15-14-08-27.bag -q -d 2 -s 3 -r 2.0"/>
+
+	<node pkg="rviz" type="rviz" name="maplab_rviz" args="-d $(find maplab_node)/share/maplab.rviz -f map"/>
+
+	<!-- OKVIS -->
+  <group ns="okvis">
+		<node name="okvis_node" pkg="okvis_ros" type="okvis_node" output="screen" required="true" args="--alsologtostderr --v=3">
+			<param name="config_filename" value="$(find maplab_launch)/config/lidarstick/calib/lidarstick-150-deg-cams-okvis.yaml"/>
+			<remap from="/camera0" to="/VersaVIS/cam1/image_raw"/>
+			<remap from="/camera1" to="/VersaVIS/cam0/image_raw"/>
+			<remap from="/imu" to="/VersaVIS/imu"/>
+			<!-- remove okvis tf frames from tf -->
+			<remap from="/tf" to="/tf_okvis"/>
+
+			<remap from="/okvis/okvis_node/okvis_maplab_odometry" to="/odometry/maplab_odom_T_M_I"/>
+		</node>
+	</group>
+
+	<!-- LIDAR SUBMAP THROTTLER -->
+	<node name="lidar_throttler" type="throttle" pkg="topic_tools"
+				args="messages /os1_node/points 0.2 /os1_node/points_throttled"
+				if="true"/>
+
+	<!-- MAPLAB -->
+	<node name="maplab_node" pkg="maplab_node" type="maplab_node" output="screen" clear_params="true">
+
+		<!-- GENERAL PARAMS -->
+		<rosparam command="load" file="$(find maplab_launch)/config/lidarstick/ros/lidarstick-maplab-node-rosparam.yaml"/>
+
+		<!-- ROBOT SPECIFIC PARAMS -->
+		<!-- SENSORS -->
+		<param name="sensor_calibration_file" value="$(find maplab_launch)/config/lidarstick/calib/lidarstick-150-deg-cams-sensors.yaml"/>
+		<param name="imu_to_camera_time_offset_ns" value="19779092"/>
```

Comment:
are you supposed to put configuration here or in the rosparam file?

Reply:
the idea was to use a few rosparam files that cover the different use-cases/projects and the set/overwrite the robot specific settings and other settings that you change often (e.g. whether or not resources should be attached) in the launch file. I think this makes it very compact and is a good trade-off between having a stable launch file for each robot and not having all these large copy-pasted launch files or param files. Wdyt?

Reply:
Agreed, makes sence. Maybe we should put that strategy somewhere...